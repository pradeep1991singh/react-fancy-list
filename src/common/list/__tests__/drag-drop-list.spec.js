import React from 'react';
import renderer from 'react-test-renderer';
import DragDropList from '../drag-drop-list';

import {
    PLANETS,
} from '../../../../__mocks__/static-data'

describe('DragDropList', () => {
    let wrapper;
    let tree;

    beforeEach(() => {
        wrapper = renderer.create(
            <DragDropList items={PLANETS} />
        );

        tree = wrapper.toJSON()
    })

    it('should render DragDropList properly', () => {
        expect(tree).toMatchSnapshot();
    });
});
