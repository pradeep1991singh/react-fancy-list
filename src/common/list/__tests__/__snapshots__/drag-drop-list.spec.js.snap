// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DragDropList should render DragDropList properly 1`] = `
<div
  className="list-container"
/>
`;
